packages/server/graphql/mutations/helpers/notifySlack.ts

```diff
@@ -97,11 +104,55 @@ export const startSlackMeeting = async (
     utm_campaign: 'invitations'
   }
   const options = {searchParams}
-  const team = await dataLoader.get('teams').load(teamId)
+  const [team, meeting] = await Promise.all([
+    dataLoader.get('teams').load(teamId),
+    dataLoader.get('newMeetings').load(meetingId)
+  ])
+  const meetingUrlWithParams = makeAppURL(appOrigin, `meet/${meetingId}`, options)
+  const meetingUrl = makeAppURL(appOrigin, `meet/${meetingId}`)
```

Comment:
To keep the link shortish, I've removed the utm params. Hopefully, this isn't too problematic for our analytics?